gocd/generated-pipelines/relay-next-us.yaml

```
@@ -0,0 +1,175 @@
+format_version: 10
+pipelines:
+  region-deploy-relay-next-us:
+    environment_variables:
+      SENTRY_REGION: us
+    group: relay-next-region-deployments
+    lock_behavior: unlockWhenFinished
+    materials:
+      relay_repo:
+        branch: master
+        destination: relay
+        git: git@github.com:getsentry/relay.git
+        shallow_clone: true
+      upstream_pipeline:
+        pipeline: deploy-relay-next
+        stage: pipeline-complete
+    stages:
+      - ready:
+          jobs:
+            ready:
+              tasks:
+                - exec:
+                    command: true
+      - wait:
+          approval:
+            type: manual
+          jobs:
+            wait:
+              tasks:
+                - exec:
+                    command: true
+      - checks:
+          fetch_materials: true
+          jobs:
+            checks:
+              elastic_profile_id: relay
+              environment_variables:
+                GITHUB_TOKEN: '{{SECRET:[devinfra-github][token]}}'
+              tasks:
+                - script: |
+                    ##!/bin/bash
+
+                    /devinfra/scripts/checks/githubactions/checkruns.py \
+                        getsentry/relay \
+                        "${GO_REVISION_RELAY_REPO}" \
+                        "Integration Tests" \
+                        "Test (macos-latest)" \
+                        "Test (windows-latest)" \
+                        "Test All Features (ubuntu-latest)" \
+                        "Push GCR Docker Image"
+              timeout: 1800
+      - deploy-production:
+          fetch_materials: true
+          jobs:
+            create_sentry_release:
+              elastic_profile_id: relay
+              environment_variables:
+                SENTRY_AUTH_TOKEN: '{{SECRET:[devinfra-temp][relay_sentry_auth_token]}}'
+                SENTRY_ORG: sentry
+                SENTRY_PROJECT: relay
+                SENTRY_URL: https://sentry.my.sentry.io/
+              tasks:
+                - script: |
+                    ##!/bin/bash
+
+                    ./relay/scripts/create-sentry-release "${GO_REVISION_RELAY_REPO}"
+              timeout: 1200
+            deploy:
+              elastic_profile_id: relay
+              tasks:
+                - script: |
+                    ##!/bin/bash
+
+                    eval $(/devinfra/scripts/regions/project_env_vars.py --region="${SENTRY_REGION}")
+
+                    /devinfra/scripts/k8s/k8stunnel
+
+                    /devinfra/scripts/k8s/k8s-deploy.py \
+                        --label-selector="service=relay,deploy_if_production=true" \
+                        --image="us.gcr.io/sentryio/relay:${GO_REVISION_RELAY_REPO}" \
+                        --container-name="relay"
+              timeout: 1200
+      - deploy-pops:
+          fetch_materials: true
+          jobs:
```

```suggestion
      - deploy-pops:
          approval:
            type: manual
          fetch_materials: true
          jobs:
```

Is it possible to add here `approve` to `manual`? Or this step by default will have to be approved?

---

Sorry that I missed the manual approval in relay-pops.

I've added an extra stage after deploy-production, `progress-to-pops` that is a manual approval step. This will be a manual approval for each region, which I'm assuming is the best option for now, however if you wanted to to just be for US deployments let me know and I can make that change.